### PR TITLE
Change namespace overview icon

### PR DESCRIPTION
**Pull request by maciaszczykm**
https://github.com/kubernetes/dashboard/pull/2497#issuecomment-338814003

### CODE REVIEW

**Comment by maciaszczykm** [comment]
@cheld Please take a look.

**Comment by cheld** [comment]
So it now this icon

https://material.io/icons/#ic_description

**Comment by cheld** [comment]
hm...I will also have a look

**Comment by cheld** [comment]
https://material.io/icons/#ic_explore

**Comment by cheld** [comment]
In case we consider it like a bookmark (I have not tested the code)
https://material.io/icons/#ic_bookmark_border

**Comment by maciaszczykm** [comment]
I like compass icon, bookmark not so much. However, I still think that we should follow @rahuldhide suggestions here. Mainly because we should try to be consistent with other pages around the web.

**Comment by cheld** [comment]
@rahuldhide suggested a view-kind-of-icon, while description is a document.

**Comment by cheld** [comment]
Ok, I tested the functionality. It actually switches the namespace selector to the current namespaces and shows all resources of that namespace. So my proposal would be

Icon: https://material.io/icons/#ic_open_in_browser
Label: Show
Tooltip: "Show all resources of this namespace"

**Comment by maciaszczykm** [comment]
@cheld `Show` is too generic.

**Comment by cheld** [comment]
Thats complicated :-(
overview1
overview2
overview3
overview4

File: src/app/frontend/common/components/actionbar/actionbarnamespaceoverview.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 <md-button ng-click="$ctrl.goToNamespaceOverview()"
            class="kd-actionbar-button">
-  <md-icon>public</md-icon>
+  <md-icon>description</md-icon>
   <md-tooltip md-delay="500"
               md-autohide>
     [[Go to namespace overview page|]]
```
Comment (cheld) [inline src/app/frontend/common/components/actionbar/actionbarnamespaceoverview.html]: Please add descriptions

**Comment by maciaszczykm** [comment]
`Select` should only select namespace, currently we redirect to overview.

`Show` is too generic.

`Resources` is a bit more like `Overview`, I like second more as it is visible in the menu and user can expect what will happen.

`Open` is a interesting idea.

File: src/app/frontend/common/components/actionbar/actionbarnamespaceoverview.html (hunk 1 above)
Comment (maciaszczykm) [reply src/app/frontend/common/components/actionbar/actionbarnamespaceoverview.html]: I will.

**Comment by cheld** [comment]
@maciaszczykm please try 'select' and 'open' a couple of times in the application. The experience is a bit different when using than just looking at static content.

**Pull request merged**